editor/src/components/inspector/widgets/ui-grid-row.tsx

```diff
@@ -86,7 +86,8 @@ export const UIGridRow: React.FunctionComponent<GridRowProps> = ({
     style={{
       padding: padded ? `0px ${UtopiaTheme.layout.rowHorizontalPadding}px` : undefined,
       display: 'grid',
-      height: tall ? UtopiaTheme.layout.rowHeight.max : UtopiaTheme.layout.rowHeight.normal,
+      minHeight: tall ? UtopiaTheme.layout.rowHeight.max : UtopiaTheme.layout.rowHeight.normal,
+      whiteSpace: 'normal',
```

Comment:
NB: This is a behaviour change. It allows for longer copy in rows and will make them expand automatically.

 The only place where that's an issue is rows with extra-long labels which before would get truncated, and now can span multiple lines. As that's the exception, happy with this change for the default behaviour.